package.json

```diff
@@ -49,7 +49,7 @@
     "@vitest/coverage-v8": "^3.2.4",
     "prettier": "^3.5.3",
     "turbo": "^2.5.4",
-    "vitest": "^3.1.1"
+    "vitest": "^3.2.4"
```

Comment:
Tests weren't running because Vitest and vitest/coverage versions weren't matching